src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -176,12 +176,14 @@ export interface RestorationFilingIF {
     foundingDate: string
   }
   restoration: {
+    approvalType?: ApprovalTypes
```

Comment:
Why is this conditional?

Reply:
Why is courtOrder optional (since it's always assigned in the filing JSON in filing-template-mixin?

Ditto the other optional fields.